### DISCUSSION TITLE
May I ask how to set the minimum version of macOS. How to directly generate app package structure?

### DISCUSSION THREAD

**Discussion by pcevent** [slint-ui/slint#3139] [Q&A] [answered]
May I ask how to set the minimum version of macOS. How to directly generate app package structure?

**Comment by tronical** [comment, marked as answer]
I haven't done this myself yet, but I think that you can use [cargo-bundle](https://crates.io/crates/cargo-bundle). We use `cargo-bundle` to create the bundle for the LSP that we ship with the VS code extension - that's similar.

The minimum version of macOS for deployment is specified using the `MACOSX_DEPLOYMENT_TARGET` environment variable.

**Comment by pcevent** [comment]
Okay, thank you!
